controllers/bsl.go

```diff
@@ -347,3 +352,15 @@ func (r *DPAReconciler) ensureBackupLocationHasVeleroOrCloudStorage(dpa *oadpv1a
 	}
 	return nil
 }
+
+func (r *DPAReconciler) resticCustomCAEnabled(bslSpec velerov1.BackupStorageLocationSpec, dpa *oadpv1alpha1.DataProtectionApplication) (bool, error) {
+	insecureSkipTLSVerify, skipPresent := bslSpec.Config["insecureSkipTLSVerify"]
+	if !skipPresent || insecureSkipTLSVerify != "false" {
+		return false, nil
+	}
+	if bslSpec.ObjectStorage.CACert == nil {
```

Comment:
Maybe to simplify things, we need to add another condition like if `insecureSkipTLSVerify` == `false` && `bslSpec.ObjectStorage.CACert == nil`